silkrpc/ethbackend/backend_grpc.cpp

```diff
@@ -93,21 +93,29 @@ asio::awaitable<PayloadStatus> BackEndGrpc::engine_new_payload_v1(ExecutionPaylo
     EngineNewPayloadV1Awaitable npc_awaitable{executor_, stub_, queue_};
     auto req{encode_execution_payload(payload)};
     const auto reply = co_await npc_awaitable.async_call(req, asio::use_awaitable);
-    PayloadStatus payload_status;
-    payload_status.status = decode_status_message(reply.status());
-    // Set LatestValidHash (if there is one)
-    if (reply.has_latestvalidhash()) {
-        payload_status.latest_valid_hash = bytes32_from_H256(reply.latestvalidhash());
-    }
-    // Set ValidationError (if there is one)
-    const auto validation_error{reply.validationerror()};
-    if (validation_error != "") {
-        payload_status.validation_error = validation_error;
-    }
+    PayloadStatus payload_status = decode_payload_status(reply);
     SILKRPC_DEBUG << "BackEnd::engine_new_payload_v1 data=" << payload_status << " t=" << clock_time::since(start_time) << "\n";
     co_return payload_status;
 }
 
+asio::awaitable<ForkchoiceUpdatedReply> BackEndGrpc::engine_forkchoice_updated_v1(ForkchoiceUpdatedRequest forkchoice_updated_request) {
+    const auto start_time = clock_time::now();
+    EngineForkChoiceUpdatedV1Awaitable fcu_awaitable{executor_, stub_, queue_};
+    auto req{encode_forkchoice_updated_request(forkchoice_updated_request)};
```

Comment:
Maybe `req` can be declared `const` here?

Reply:
yeah